Guide/testing.markdown

```diff
@@ -163,3 +170,62 @@ tests = aroundAll (withIHPApp WebApplication config) do
 
 ## Advanced
 For more details on how to structure test suites see the [Hspec manual](http://hspec.github.io/) (a Haskell testing library). You also might want to check out the cool [Hedgehog](https://hedgehog.qa/) library for automated property tests.
+
+## GitHub Actions
+
+The following GitHub Action workflow can be used to run the tests on CI:
+
+```yaml
+# .github/workflows/test.yml
+
+name: Test
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  tests:
+    name: Run Tests
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: cachix/install-nix-action@v20
+      with:
+        nix_path: nixpkgs=https://github.com/NixOS/nixpkgs/archive/51bcdc4cdaac48535dabf0ad4642a66774c609ed.tar.gz
+
+    # Use the cachix cache for faster builds.
+    - name: Cachix Init
+      uses: cachix/cachix-action@v12
+      with:
+        name: digitallyinduced
+        skipPush: true
```

Comment:
Maybe can be removed, as I think only you have write access?

Reply:
`skipPush: true` needs to set to make it a read-only cache. Otherwise it will try to push any new binaries nix created at the end of the build (which will fail due to missing write access)